test/e2e/scalability/load.go

```diff
@@ -703,3 +720,30 @@ func CreateNamespaces(f *framework.Framework, namespaceCount int, namePrefix str
 	}
 	return namespaces, nil
 }
+
+func CreateQuotas(f *framework.Framework, namespaces []*v1.Namespace, podCount int, testPhase *timer.Phase) error {
+	defer testPhase.End()
+	quotaTemplate := &v1.ResourceQuota{
+		Spec: v1.ResourceQuotaSpec{
+			Hard: v1.ResourceList{"pods": *resource.NewQuantity(int64(podCount), resource.DecimalSI)},
+		},
+	}
+
+	for _, ns := range namespaces {
+		var got *v1.ResourceQuota
+		if err := wait.PollImmediate(2*time.Second, 30*time.Second, func() (bool, error) {
+			var err error
+			quotaTemplate.Name = ns.Name + "-quota"
+			got, err = f.ClientSet.CoreV1().ResourceQuotas(ns.Name).Create(quotaTemplate)
```

Comment:
_, err

Reply:
Why do you need to capture the resourcequota?

Reply:
Right. I don't need it anymore.